server/index.js

```diff
@@ -89,20 +88,37 @@ var server = app.listen(port, () => {
   console.log(`Listening on ${port}`);
 });
 
-app.use(morgan("tiny"));
-app.use('/uploads', express.static('uploads'));
 
 //Default path
 app.get("/", (req, res) => {
   res.send("Hello World Test!");
 });
 
+//path POST /upload
+app.post("/uploads", upload.single("myImage"), (req, res) => {
+  console.log(`File: ${req}`);
+  res.json({ file: req.file });
+
+
 //path POST /upload
```

Comment:
This functions just fine but before the sprint 4 completion date this should follow the same format as the other routes, meaning the handling should be moved out of index.js and into the routes directory. Just because part of the grading scheme is following the same conventions throughout.

Reply:
Also, just remember to reference issue numbers for every commit. Otherwise the commits don't get linked to the issue and anyone debugging in the future won't know how we solved the issue. 

Otherwise, great work & thank god photos can be displayed.

Reply:
Yes I agree, I will try and change the convention to the same as the other routes. I also keep forgetting to reference every commit. I gotta get on top of that. Thanks for the feedback